templates/prometheus.go

```diff
@@ -136,31 +106,10 @@ var PrometheusTemplate = promv1.Prometheus{
 				},
 			},
 		},
-		RemoteWrite: []promv1.RemoteWriteSpec{
-			{
-				OAuth2: &promv1.OAuth2{
-					ClientSecret: corev1.SecretKeySelector{
-						LocalObjectReference: corev1.LocalObjectReference{},
-					},
-					ClientID: promv1.SecretOrConfigMap{
-						Secret: &corev1.SecretKeySelector{
-							LocalObjectReference: corev1.LocalObjectReference{},
-						},
-					},
-					EndpointParams: map[string]string{},
-				},
-				WriteRelabelConfigs: []promv1.RelabelConfig{
-					{
-						SourceLabels: []string{"__name__", "alertname"},
-						Regex:        getRelableRegex(alerts, metrics),
-						Action:       "keep",
-					},
-				},
-			},
-		},
```

Comment:
Please put this section in a comment instead of removing it. We will need to reintroduce it very soon.

Reply:
Added it back in the commit where is was removed